buildlib/tools/builds.sh

```diff
@@ -385,7 +385,7 @@ build_static() {
 	$MAKEP
 	$MAKEP install
 
-	$MAKE -C test/apps/uct_info
+	PKG_CONFIG_PATH=$ucx_inst/lib/pkgconfig $MAKE -C test/apps/uct_info LDFLAGS="-Wl,-rpath=$ucx_inst/lib"
```

Comment:
1. IMO would be more readable to export before make, and unset after
2. Maybe better export LD_LIBRARY_PATH (when running uct_info non static) instead of forcing rpath?

Reply:
> * IMO would be more readable to export before make, and unset after

hmmm, not sure it is better readable, but ok, done

> * Maybe better export LD_LIBRARY_PATH (when running uct_info non static) instead of forcing rpath?

ok, done